src/componentsEventPage/CategoryFilter.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+
+export default function CategoryFilter ({ onSelectFilter }) {
+  
+  return (
+     <FilterContainer>
+      <CategoryButton name="metal"  onClick={() => onSelectFilter('metal')}>metal</CategoryButton>
+      <CategoryButton name="medieval" onClick={() => onSelectFilter('medieval')}>medieval</CategoryButton>
+      <CategoryButton name="sand"  onClick={() => onSelectFilter('sand')}>sand</CategoryButton>  
+      <CategoryButton name="others"  onClick={() => onSelectFilter('others')}>others</CategoryButton>
+      </FilterContainer>
+  )
+  
+}  
+
+const FilterContainer = styled.div `
+`
+
+const CategoryButton = styled.button `
+display: inline-block;
+    font-weight: 200;
+    height: 25px;
+    width: 95px;
```

Comment:
Try to make the buttons adjust to their content when it comes to size. Having fixed `width` and `height` is not very flexible. What if one of your categories will have a very long name?

Reply:
This is still a placeholder, because my design is not integrated yet